README.md

```diff
@@ -41,6 +41,20 @@ jdbcUrl==jdbc:jtds:sqlserver://server\;instance=inst\;domain=dmn\;useNTLMv2=true
 * **zorderby_col**          `optional` Columns on which perform z-ordering, viz [docs](https://docs.delta.io/2.0.0/optimizations-oss.html#z-ordering-multi-dimensional-clustering). Note, the effectiveness of the locality drops with each extra column.
 * **log_lvl**               `optional` Default log level is info. Log level may be `"off, error, warn, info, debug, trace, all"`.
 * **log_spark_lvl**         `optional` Default log level is info. Log level may be `"ALL", "DEBUG", "ERROR", "FATAL", "INFO", "OFF", "TRACE", "WARN"`.
+* **CT_changes_query**            `optional` Custom query to get current Change Tracking version, e.g. `exec dbo.getCTCurrentVersion`.
```

Comment:
please use lowercase for arguments

Reply:
Should it be changed in `Config` too, e.g. [here](https://github.com/Temabit-FOZZY-Group/mirroring/blob/e5e9d306fa7ecaf08f66a795523f2ec0e2afe4ea/src/main/scala/palefat/data/mirror/Config.scala#L65)?

Reply:
I don't think it has to do with coding style
I meant that the arguments we had before were all in lowercase and I think this is convenient

Reply:
Fixed.